README.md

```diff
@@ -20,9 +20,13 @@ Or if your project uses [`poetry`](https://python-poetry.org/) you can add it to
 poetry add yapapi
 ```
 
-### Local development setup
+## API Reference
 
-#### Poetry
+For a comprehensive API reference, please refer to [our official readthedocs page](https://yapapi.readthedocs.io/).
```

Comment:
I can't reach that far up, but we can also add a badge for `readthedocs` next to all the other badges (see: https://docs.readthedocs.io/en/stable/badges.html).

Reply:
added (locally for now)